pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[project]
+name = "PatientX"
+version = "0.1.0"
+
+license = { text = "MIT" }
+
+readme = "README.md"
+
+dependencies = [
+    "bertopic",
+    "datamapplot",
+    "numpy",
+    "pandas",
+    "spacy",
+]
+
+[project.scripts]
+my_tool = "repo.run:main"
+
+[project.urls]
+Homepage = "https://github.com/gt-sse-center/PatientX.AI"
```

Comment:
Add a note so that you remember to update these URLs when the repository is transferred to the PI.